src/include/sof/math/log.h

```diff
@@ -0,0 +1,21 @@
+/* SPDX-License-Identifier: BSD-3-Clause
+ *
+ * Copyright(c) 2021 Intel Corporation. All rights reserved.
+ *
+ * Author: Shriram Shastry <malladi.sastry@linux.intel.com>
+ *
+ */
+
+#ifndef __SOF_MATH_BASE_LOGARITHM_H__
+#define __SOF_MATH_BASE_LOGARITHM_H__
+
+#include <stdint.h>
+
+#define ONE_OVER_LOG2_10	0x268826A2000ULL	/* 1/log2(10) */
```

Comment:
Please add to comment the used Q format for these constants, this and next. I guess the first is Q1.43. But I don't get exactly the same hex number if I try to calculate it.

Reply:
It was a very small difference but thanks for the inputs.